rkt/app_start.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	pkgPod "github.com/coreos/rkt/pkg/pod"
+	"github.com/coreos/rkt/stage0"
+
+	"github.com/appc/spec/schema/types"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdAppStart = &cobra.Command{
+		Use:   "start UUID --app=NAME",
+		Short: "Start an app in a pod",
+		Long:  `Start appz!`,
+		Run:   runWrapper(runAppStart),
+	}
+)
+
+func init() {
+	cmdAppStart.Flags().StringVar(&flagAppName, "app", "", "app to start")
+	cmdApp.AddCommand(cmdAppStart)
+}
+
+func runAppStart(cmd *cobra.Command, args []string) (exit int) {
+	if len(args) < 1 {
+		stderr.Print("must provide the pod UUID")
+		return 1
+	}
+
+	if flagAppName == "" {
```

Comment:
We could, that sounds like a nice addition. If we intend the `app` subcommands to be first-class for end-users, I'm all for this feature. For the current use-case of machine-only usage (rktlet) I'd say it is even good to err out.